Ukol7,8,9

DominikHavelunob opened the pull request:
ukol 7,8,9 + unittests by mely byt overeny

TomasRacil committed:
Ukol 3 added and tests modified

DominikHavelunob committed:
Update sifry.cpp

DominikHavelunob committed:
Update sifry.cpp

TomasRacil committed:
Update README.md

TandRCZ committed:
Ukol 0 vyresen (#44)

TomasRacil committed:
Oprava nazvu testu

TomasRacil committed:
Merge branch 'main' of https://github.com/TomasRacil/Ukoly2024

TomasRacil committed:
PostMergeFix

TomasRacil commented on Ukol_2/cpp/sifry.cpp:
```diff
@@ -1,47 +1,228 @@
+    file.close();  
+
+  }else{
+    file << obsah << endl;
```
zde je navíc endl pridavate tak vzdy prazdny radek a konec i kdyby nebyl v retezci pro ulozeni

TomasRacil commented on Ukol_2/cpp/sifry.cpp:
```diff
@@ -1,47 +1,228 @@
+        }
+      }  
+      string str(charArray);
+      return charArray;
```
doporucuji vracet primo string pokud vracite pole znaku je potreba ho ukoncit \0 (ukoncovacim znakem)

TomasRacil commented on Ukol_2/cpp/sifry.cpp:
```diff
@@ -1,47 +1,228 @@
+        }
+      }
+      string str(charArray);
+      return charArray;
```
to same zde

TomasRacil commented on Ukol_2/cpp/sifry.cpp:
```diff
@@ -1,47 +1,228 @@
 }
 
 // Funkce pro XOR šifru
 std::string xor_sifra(const std::string &text, const std::string &klic, bool sifrovat)
```
XOR mi taky nevraci uplně ty spravne hodnoty ale tam mě to nepřekvapuje hodně zalezi na implementaci teto funkce.

TomasRacil reviewed (changes requested):
Prosim ještě upravit zminěne

TomasRacil committed:
opravy testů

TomasRacil committed:
TEST sync

TomasRacil committed:
Ukol 4 přidán

TomasRacil committed:
Ukol 4 fix

DominikHavelunob committed:
Merge branch 'TomasRacil:main' into main

DominikHavelunob committed:
linked_list.cpp

DominikHavelunob committed:
hello_world.cpp

DominikHavelunob committed:
vypocty.cpp

TomasRacil committed:
Ukol 5 pridan

DominikHavelunob committed:
advent_lod.cpp

TomasRacil commented on Ukol_3/cpp/linked_list.cpp:
```diff
@@ -0,0 +1,207 @@
+    Node *temp = head;
+    while (temp != nullptr)
+    {
+        os << temp->data << " ";
```
U tohoto vám test neprochází test protože přídáváte mezeru i nakonec tedy "1 2 3 " místo očekávaného "1 2 3"

TomasRacil reviewed (changes requested):
u úkolů 0 a 1 není implementováno řešení v Pythonu u druhého úkolu XOR nevrácí v osmičkové soustavě a u 3 úkolu je dorbý problém s výpisem linked listu

TomasRacil committed:
Update testCpp.yml

TomasRacil committed:
Úkol 6 pridan

DominikHavelunob committed:
Merge branch 'TomasRacil:main' into main

DominikHavelunob committed:
Update hanoi.cpp

TomasRacil committed:
Ukol 7 pridan

TomasRacil committed:
Ukol 8 pridan opraven test ukolu 7

TomasRacil committed:
Ukol 9 pridan

TomasRacil committed:
Ukol 9 workflow

TomasRacil committed:
Ukol 9 aktualizace zadani

DominikHavelunob committed:
Merge branch 'TomasRacil:main' into main

DominikHavelunob committed:
Update matice.py

DominikHavelunob committed:
Update matice.py

DominikHavelunob committed:
Update Knihovna.py

DominikHavelunob committed:
Update Kniha.py

DominikHavelunob committed:
Update Ctenar.py

DominikHavelunob committed:
Update matice.py

DominikHavelunob committed:
Update matice.py

DominikHavelunob committed:
Update matice.py

DominikHavelunob committed:
Update hello_world.cpp

DominikHavelunob committed:
Update hello_world.py

DominikHavelunob committed:
Update vypocty.py

DominikHavelunob committed:
Update sifry.cpp

DominikHavelunob committed:
Update sifry.cpp

DominikHavelunob committed:
Update sifry.cpp

DominikHavelunob committed:
Update linked_list.cpp

verze 2

DominikHavelunob committed:
Update linked_list.cpp

zmena z pointer to pointer na pointer u deleteAtIndex

DominikHavelunob committed:
Update linked_list.cpp

navraceni pointeru zpet na p2p

DominikHavelunob committed:
Update linked_list.cpp

pridano pretizeni

DominikHavelunob committed:
Update linked_list.cpp

oprava mezery v ostream& operator<<

DominikHavelunob committed:
Update advent_lod.cpp

reupload

DominikHavelunob committed:
Update advent_lod.cpp

zmena commentu

DominikHavelunob committed:
Update hanoi.cpp

reupload

DominikHavelunob committed:
Update hanoi.cpp

zmena v zobrazVeze

DominikHavelunob committed:
Update matice.py

reupload

DominikHavelunob committed:
Update matice.py

reupload

DominikHavelunob committed:
Update main.py

reupload

DominikHavelunob committed:
Update Ctenar.py

reupload

DominikHavelunob committed:
Update Kniha.py

reupload

DominikHavelunob committed:
Update Knihovna.py

reupload

DominikHavelunob committed:
Update hanoi.cpp

DominikHavelunob committed:
Update test.cpp

DominikHavelunob committed:
Update hanoi.cpp

DominikHavelunob committed:
Update test.cpp

DominikHavelunob committed:
Update hanoi.cpp

DominikHavelunob committed:
Create main.cpp

DominikHavelunob committed:
Delete Ukol_5/cpp/main.cpp

DominikHavelunob committed:
uploaded old ecosystem project

build command: g++ -std=c++17 entity.cpp organism.cpp plant.cpp animal.cpp herbivore.cpp carnivore.cpp environment.cpp main.cpp
eqilibrium neni specificke kvuli randomizaci hodnot pro spawn entit
